apps/mobile-wallet/ios/Podfile.lock

```diff
@@ -1618,7 +1618,7 @@ PODS:
     - React-Core
   - RNFlashList (1.6.4):
     - React-Core
-  - RNGestureHandler (2.16.2):
+  - RNGestureHandler (2.21.2):
     - DoubleConversion
```

Comment:
Contact address "dot" and internal address "dot" have diff sizes

<img width="350" alt="image" src="https://github.com/user-attachments/assets/375d47c2-c871-440e-991e-7f89bf3ecacd">

Reply:
Solved in https://github.com/alephium/alephium-frontend/pull/1032